cves/kernel/CVE-2018-13406.yml

```diff
@@ -399,8 +430,10 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      Here, the integer value could have been such that it exceeds the maximum INT size. 
+      Inputs, even from other feeder functions should not be trusted and checks should be done to ensure they don't pose a threat.
   security_by_obscurity:
     applies:
```

Comment:
I would mark these as false instead of leaving them blank, but that might be picky of me.

Reply:
I was hesitant to do so, as in the future someone might come up with an explanation and prove that another principle also applies.